rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.1.0-release-notes.adoc

```diff
@@ -0,0 +1,34 @@
+= RPA Recorder 6.1.0 Release Notes
+
+RPA Recorder is MuleSoft’s tool for recording a business process for automation.
+
+== TBD
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The new attributes selection panel enables you to fine-tune the XPath for web elements.
+* You can now capture _Mouse Over_ events using the `F2` key.
+* You can now activate or deactivate hotkeys.
+* The new *Open log file* button enables you to open the latest log file.
+* The new *Open log files directory* enables you to open the folder containing all RPA Recorder logs in your local computer.
```

Comment:
Add additional information "...button in settings -> Log files" - there is a new Tab in the Settings called "Log files" where this button is found.